action.yml

```diff
@@ -53,6 +53,14 @@ inputs:
   image_family:
     description: The image family for the operating system that the boot disk will be initialized with.
     required: false
+  preemptible:
+    description: Use GCE preemptible VM instance; https://cloud.google.com/compute/docs/instances/preemptible
+    default: true
```

Comment:
Should `preemptible` be `true` by default? I think I would be more comfortable with default `false`?

Reply:
Up to you.

Reply:
@Manouchehri +1 to default false please